json_handler.go

```diff
@@ -42,9 +42,6 @@ func (h *JSONHandler) clear() {
 
 // TryHandle tells if this line was handled by this handler.
 func (h *JSONHandler) TryHandle(d []byte) bool {
-	if !bytes.Contains(d, []byte(`"time":"`)) {
```

Comment:
Actually, I think making this be `"time":` would solve the problem and avoid wasting time trying to parse 99% of non-JSON lines.